apps/look-and-feel-stories/src/AccordionCore/AccordionCore.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import { AccordionCore } from "@axa-fr/design-system-apollo-react";
+import { Meta, StoryObj } from "@storybook/react";
+import type { ComponentProps } from "react";
+
+const meta: Meta<typeof AccordionCore> = {
+  component: AccordionCore,
+  title: "Components/AccordionCore (LF)",
```

Comment:
Pas besoin de préciser, c'est dans un storybook dédié à l&f
```suggestion
  title: "Components/AccordionCore",
```

Reply:
Juste pour mettre en évidence la différence entre l'ancien et le nouveau composant, puisque j'ai gardé les deux stories

Reply:
C'est une bonne idée de base le problème c'est que cela crée un conflit de css
![image](https://github.com/user-attachments/assets/0f38820a-33fb-44d9-8698-3cc1d567af25)

Ce que je propose c'est de garder le code de l'ancien accordéon (pas de break pour les projet qui l'utilise)
juste mettre un deprecated

Pour les stories on garde que les nouvelle stories avec dans le mdx un encadrer d'information (on peut utilisait le composant Message en mode warning en plus 😉) pour indique que l'ancien import et déprécier.



Qu'en pensez vous ?
@ousmanw-axa @GuillaumeKESTEMAN

Reply:
Oui, on garde l'ancien code en mode deprecated.
et pour les stories, on utilise celui d'Appollo/LF et on supprime les anciens story deprecated.

Reply:
Oui, je suis d'accord 👍